docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -190,7 +190,10 @@ Once you have properly configured secure user roles and permissions, to avoid ex
 
 - Avoid exposing servers on the open internet when not strictly required.
 - Ensure any servers, services, ports and virtual hosts that were opened for testing purposes are shut down before going live.
-- Secure the database with a good password, keys, firewall, etc. Ensure that the database user used by the web app only has access to do the operations needed by [[= product_name =]]. The Data Definition Language (DDL) commands (create, alter, drop, truncate, comment) are not needed for running [[= product_name =]], only for installing and upgrading it. If the web app user does not have these rights, then that reduces the damage that can be done if there is a security breach.
+- Ensure file system permissions are set up such that the web server or PHP user can't access files it shouldn't be able to read.
```

Comment:
```suggestion
- Ensure file system permissions are set up in such way that the web server or PHP user can't access files it shouldn't be able to read.
```

Reply:
Ok with me, but then "in such **a** way" please.
```suggestion
- Ensure file system permissions are set up in such a way that the web server or PHP user can't access files it shouldn't be able to read.
```